src/work/worker.ts

```diff
@@ -368,13 +376,15 @@ export class Worker {
 					? 'ffmpeg.exe'
 					: 'ffmpeg',
 				'-hide_banner',
+				targetTime ? `-ss ${targetTime}` : undefined,
```

Comment:
Will FFmpeg fall back to the last frame or something like that when, for whatever reason, trying to seek past the actual duration?

Reply:
Addressed by https://github.com/olzzon/tv-automation-media-management/pull/1/commits/b80dbe1cc655287868c047d9e9a19d3a670a8807